code/modules/projectiles/ammo_datums.dm

```diff
@@ -1876,7 +1876,7 @@
 	accurate_range = 10
 	damage =  50
 	penetration = ARMOR_PENETRATION_TIER_6
-	accuracy = HIT_ACCURACY_TIER_9 + HIT_ACCURACY_TIER_5 // 75 accuracy
+	accuracy = -HIT_ACCURACY_TIER_2 // 75 accuracy
```

Comment:
I feel like half the PRs you make I have to do some mind wrapping to get to the point where things are considered a "fix". 

Remove the comment as it's not relevant to the total configuration of the weapon (as multiple guns with various settings could potentially use any ammo datum).

This is a balance change (not one I'm against mind you). 

Please stop tagging these as fixes.

Reply:
Sorry about that. Will keep in mind going forward